kytos/core/auth.py

```diff
@@ -368,6 +368,8 @@ def _update_user(self, username):
             updated = self.user_controller.update_user(username, data)
         except ValidationError as err:
             raise BadRequest from err
+        except DuplicateKeyError as err:
+            raise Conflict from err
```

Comment:
Missed this error from `user_controller.update_user()`

Reply:
Good catch, Aldo.